vnext/Microsoft.ReactNative/Modules/AppThemeModuleUwp.cpp

```diff
@@ -30,26 +37,27 @@ AppTheme::AppTheme(
     const std::shared_ptr<facebook::react::MessageQueueThread> &defaultQueueThread)
     : m_context(&context), m_queueThread(defaultQueueThread) {
   if (auto currentApp = xaml::TryGetCurrentApplication()) {
-    m_isHighContrast = m_accessibilitySettings.HighContrast();
+    m_isHighContrast = getIsHighContrast();
     m_highContrastColors = getHighContrastColors();
 
-    m_highContrastChangedRevoker =
-        m_accessibilitySettings.HighContrastChanged(winrt::auto_revoke, [this](const auto &, const auto &) {
-          folly::dynamic eventData = folly::dynamic::object("highContrastColors", getHighContrastColors())(
-              "isHighContrast", getIsHighContrast());
-
-          fireEvent("highContrastChanged", std::move(eventData));
-        });
+    if (react::uwp::IsWinUI3Island()) {
+      m_wmSubscription =
+          SubscribeToWindowMessage(context.Notifications(), WM_THEMECHANGED, [this](const auto &, const auto &) {
```

Comment:
>this [](start = 78, length = 4)

It used to be before, but capturing this doe snot seem to be safe.

Reply:
#7147